firestore/src/android/collection_reference_android.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2020 Google LLC
+
 #ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_ANDROID_COLLECTION_REFERENCE_ANDROID_H_
 #define FIREBASE_FIRESTORE_CLIENT_CPP_SRC_ANDROID_COLLECTION_REFERENCE_ANDROID_H_
```

Comment:
⚠️ Lint warning: `Lines should be <= 80 characters long`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/src/android/collection_reference_android.h:4:  Lines should be <= 80 characters long  [whitespace/line_length] [2]"></hidden>